prov/sockets/src/sock_conn.c

```diff
@@ -406,6 +406,8 @@ int sock_conn_listen(struct sock_ep_attr *ep_attr)
 	return -FI_EINVAL;
 }
 
+/* do NOT return negative value from this function, because
+ * its value is used as an argument to the strerror() function  */
```

Comment:
Almost all of the functions return 0 on success or negative value on error.  We should strive to be consistent with that.  I would rather change where the return value is passed to strerror().

Reply:
Yes, I agree. Applied your comment.